clients/go/cmd/zbctl/internal/commands/createWorker.go

```diff
@@ -124,8 +125,9 @@ func handle(jobClient worker.JobClient, job entities.Job) {
 		failJob(jobClient, job, stderr.String())
 	}
 
+	atomic.AddInt64(&jobsHandled, 1)
 	jobsHandled++
```

Comment:
:x: I think this should have been removed; we are already incrementing atomically.